docs/source/installation/index.md

```diff
@@ -71,10 +72,10 @@ In this example, we will set up the `localhost`, the computer where AiiDA itself
 To set up a computer, use the ``verdi`` CLI of ``aiida-core``.
 
 ```console
-$ verdi computer setup -n -L localhost -H localhost -T core.local -S core.direct
+$ verdi computer setup -n -L localhost -H localhost -T core.local -S core.direct -w $AIIDA_PATH/workdir
```

Comment:
As @sphuber points out [here](https://github.com/aiidateam/aiida-quantumespresso/pull/949#discussion_r1214421241), this wouldn't actually work in case the user hasn't set the `AIIDA_PATH` variable. My [proposed solution](https://github.com/aiidateam/aiida-quantumespresso/pull/949#discussion_r1215328875) is to use `${AIIDA_PATH:-$HOME}/aiida_scratch`.

Reply:
Go for that then!